src/bindings/python/src/pyopenvino/graph/types/element_type.cpp

```diff
@@ -46,37 +48,78 @@ void regclass_graph_Type(py::module m) {
     type.attr("u32") = ov::element::u32;
     type.attr("u64") = ov::element::u64;
     type.attr("bf16") = ov::element::bf16;
-    type.attr("undefined") = ov::element::undefined;
 
+    type.def("__hash__", &ov::element::Type::hash);
     type.def("__repr__", [](const ov::element::Type& self) {
-        std::string bitwidth = std::to_string(self.bitwidth());
-        if (self == ov::element::undefined) {
-            return "<Type: '" + self.c_type_string() + "'>";
-        } else if (self.is_signed()) {
+        if (self == ov::element::f32 || self == ov::element::f64) {
+            std::string bitwidth = std::to_string(self.bitwidth());
             return "<Type: '" + self.c_type_string() + bitwidth + "'>";
         }
-        return "<Type: 'u" + self.c_type_string() + bitwidth + "'>";
+
+        return "<Type: '" + self.c_type_string() + "'>";
     });
-    type.def("__hash__", &ov::element::Type::hash);
     type.def(
         "__eq__",
         [](const ov::element::Type& a, const ov::element::Type& b) {
             return a == b;
         },
         py::is_operator());
+
+    type.def("is_static", &ov::element::Type::is_static);
+    type.def("is_dynamic", &ov::element::Type::is_dynamic);
+    type.def("is_real", &ov::element::Type::is_real);
+    type.def("is_integral", &ov::element::Type::is_integral);
+    type.def("is_integral_number", &ov::element::Type::is_integral_number);
+    type.def("is_signed", &ov::element::Type::is_signed);
+    type.def("is_quantized", &ov::element::Type::is_quantized);
     type.def("get_type_name", &ov::element::Type::get_type_name);
+    type.def("compatible",
```

Comment:
Would `merge_compatible`, or `merge_compatibility` be more clear than `compatible`? This method concerns merging only, so for a user calling for example `dir(ov_type_object)` it would be obvious from the start that he may need to use this. I feel like `compatible` may sound a bit non-descriptive.

Reply:
It is actually reflection of the C++ API (as it is mostly required), take a look at `&ov::element::Type::compatible`.